### PR TITLE
Used shared init with LinuxKit

### PR DESCRIPTION
**Pull request by justincormack**
With https://github.com/linuxkit/linuxkit/pull/2500 the init script
can be used for both bare metal and in Docker.

Signed-off-by: Justin Cormack <justin.cormack@docker.com>

### CODE REVIEW

File: examples/linuxkit.yml [hunk 1]
```diff
@@ -1,16 +1,15 @@
 kernel:
   image: "linuxkit/kernel:4.9.41"
   cmdline: "console=ttyS0"
-init:
```
Comment (riyazdf) [inline examples/linuxkit.yml]: this stanza should update to:
```
init:
    - linuxkit/init:f7d50013e8466fb96eaa8dfa96077e8af1ee9c67
    - linuxkit/runc:a1b564248a0d0b118c11e61db9f84ecf41dd2d2a
    - linuxkit/containerd:06876ceef325e49e9ba119659357768d5df89075
    - linuxkit/ca-certificates:e44b0a66df5a102c0e220f0066b0d904710dcb10
```

Comment (justincormack) [reply examples/linuxkit.yml]: ah yes thanks

Comment (justincormack) [reply examples/linuxkit.yml]: amended

**Comment by justincormack** [comment]
Updated to latest version of PR

**Pull request merged**